admin/display/menu.php

```diff
@@ -23,6 +23,10 @@ function __construct() {
 		} else {
 			return;
 		}
+
+		// enqueued here because All_in_One_SEO_Pack_Module::admin_enqueue_scripts does not work
```

Comment:
This note might be better located on the enqueue function's phpDoc.

Reply:
Done.